src/components/SearchPage/SearchPage.js

```diff
@@ -1,25 +1,46 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import SearchBar from '../SearchBar/SearchBar';
 import SearchResults from '../SearchResults/SearchResults';
-import './SearchPage.css'
-import ExchangeRates from '../DisplayUserData/DisplayUserData';
 import DisplayUserData from '../DisplayUserData/DisplayUserData';
+import { useQuery } from '@apollo/client';
+import * as gql from '../../queries/queries'
+import './SearchPage.css'
 
-const SearchPage = ({query, setQuery, queryResults, setNewPost}) => {
+const SearchPage = ({setNewPost}) => {
 
+    const [query, setQuery] = useState('')
     useEffect(() => {
         setNewPost(false);
       }, [])
 
+    const { loading, error, data } = useQuery(gql.GET_ALL_USERS);
```

Comment:
Nice job adding this query here, this makes sense to keep it contained to this component